### ISSUE TITLE
What is get_lr() meaning in the optimizer.py

### ISSUE THREAD

**Issue by kugwzk** [huggingface/pytorch-pretrained-BERT#233]
I use a Model based on BertModel, and when I use the BertAdam the learning rate isn't changed. And when I use `get_lr()`, the return result is `[0]`. And I see the length of state isn't 0, but why I get that?

**Comment by thomwolf** [comment]
You can use it to get the current learning rate of the `BertAdam` optimizer (which vary according to the schedules discussed in #195).
